src/components/LicenseRevokeModal/index.jsx

```diff
@@ -52,10 +52,11 @@ class LicenseRevokeModal extends React.Component {
       fetchSubscriptionDetails,
       fetchSubscriptionUsers,
       searchQuery,
+      subscriptionUUID,
     } = this.props;
-    const options = { userId: user.userId };
+    const options = { user_email: user.user_email };
 
-    return sendLicenseRevoke(options)
+    return sendLicenseRevoke(options, subscriptionUUID)
```

Comment:
can we add subscriptionUUID to options?

Reply:
Yes, we can add it. But we don't need it at the backend. To avoid sending any extra information to backend and keep the functions same as [here](https://github.com/edx/frontend-app-admin-portal/pull/285/files) I am doing it.

Reply:
Could we take it out of options before calling the api?

Reply:
Yes, we can do that, but doesn't that will look ugly? I think if we need to pass it through options then pass it to the API aswell

Reply:
Agree it probably makes sense to keep as a separate argument, rather than taking it out of `options` before the API.